### ISSUE TITLE
Upgrading trains to clearml

### ISSUE THREAD

**Issue by majdzr** [allegroai/clearml-server#67]
Hey,

Thanks (again) for the great product!

I'm trying to upgrade the server from _trains_ (0.16.x) to _clearml_ (0.17.x) by following the instructions [here ](https://github.com/allegroai/clearml-server#upgrading-), but I receive the following error when trying to pull and spin up the docker containers:

```
ERROR: for apiserver  manifest for allegroai/clearml:latest not found: manifest unknown: manifest unknown
ERROR: for fileserver  manifest for allegroai/clearml:latest not found: manifest unknown: manifest unknown
ERROR: for webserver  manifest for allegroai/clearml:latest not found: manifest unknown: manifest unknown
```

Your help is appreciated.

**Comment by codybum** [comment]
I am having the same issue.  There does not appear to be a clearml:latest only a specific version tag allegroai/clearml:0.17.0-46.

Unfortunately, replacing latest with 0.17.0-46 on the most recent docker-compose file does not seem to work either.

I get the following error:

Creating clearml-redis      ... done
Creating clearml-mongo      ... done
Creating clearml-elastic    ... done
Creating clearml-fileserver ... done
Creating clearml-apiserver  ... error

ERROR: for clearml-apiserver  Cannot start service apiserver: driver failed programming external connectivity on endpoint clearml-apiserver (47381a33e49c49098f03ca43a0e8df180d3cd3968c91662db162d5a0d027a241): Error starting userland proxy: listen tcp 0.0.0.0:8008: bind: address already in use

ERROR: for apiserver  Cannot start service apiserver: driver failed programming external connectivity on endpoint clearml-apiserver (47381a33e49c49098f03ca43a0e8df180d3cd3968c91662db162d5a0d027a241): Error starting userland proxy: listen tcp 0.0.0.0:8008: bind: address already in use
ERROR: Encountered errors while bringing up the project.

I have confirmed that nothing is running on these ports before the compose is started.

**Comment by jkhenning** [comment]
Hi @majdzr ,

Thanks for the report!

We'll add the `latest` tag immediately, that was an oversight, our apologies 😢 

@codybum as for your report, we'll do a complete install from scratch to try and reproduce - I'll update soon...

**Comment by jkhenning** [comment]
Updating - `latest` tag was added

**Comment by jkhenning** [comment]
@codybum I've just installed a fresh copy using the latest `docker-compose.yml` file and it works for me. I guess there must be some process left running on your machine taking up port 8008 - perhaps some docker container is still up?

**Comment by codybum** [comment]
@jkhenning you were right.  I stopped all services and was able to make the default configuration work.  

I had modified the previous compose file to make use of an Apache frontend to terminate SSL.  I needed to redirect the clearml api port to 8007.

Both the default and my configuration seem to work.  I have not tested functionality yet, but at least everything looks good on the web interface.

Thanks!
